tests/std/tests/P0768R1_spaceship_operator/test.cpp

```diff
@@ -7,6 +7,10 @@
 #include <compare>
 #include <type_traits>
 
+#ifdef __clang__
+#pragma clang diagnostic error "-Wzero-as-null-pointer-constant"
+#endif // __clang__

```

Comment:
I recommend citing your PR here, since otherwise it could be mysterious why we're enabling this warning as an error. (For similar reasons, we cite bugs when testing things.)